### PR TITLE
Feature/organization search

### PR DESCRIPTION
**Pull request by JamesSCBD**

### CODE REVIEW

File: assets/scss/_theme.scss [hunk 1]
```diff
@@ -1,92 +1,109 @@
...
+  @include transform(scale(0.3 !important));
+}
+
+/** FIlter Sec  Design  **/
+.filter_Sec {
+  margin: 15px;
+}
+.filter_Sec .custom-checkbox {
+  margin-top: 5px;
+}
```
Comment (sbilodeau) [inline assets/scss/_theme.scss:92]: can you tell what what changed between line 1- 92?

File: components/list/OrganizationsList.vue [hunk 2]
```diff
@@ -53,47 +48,83 @@
...
-  { key: 'MemberCount', label: 'Member Count' },
-  // { key: 'EventCount', label: '# of Events' },
-  { key: 'Country', label: 'Country', class: 'text-center' },
-  { key: 'Score', label: 'Rank' },
+const baseColumns = [
+  { key: 'OrganizationName', label: 'Organization', sortable: true },
+  { key: 'OrganizationAcronym', label: 'Acronym', sortable: true },
+  { key: 'OrganizationTypeUID', label: 'Type', sortable: true },
+  { key: 'Government', label: 'Government', sortable: true },
+  { key: 'IsValidated', label: 'IsValidated', sortable: true },
```
Comment (sbilodeau) [inline components/list/OrganizationsList.vue:60]:
```suggestion
  { key: 'IsValidated', label: 'Validated', sortable: true },
```

File: components/list/columns/OrganizationType.vue [hunk 3]
```diff
@@ -5,14 +5,18 @@
...
 <script>
+import { mapGetters } from 'vuex';
 
 export default {
-  components: { },
-  props     : { organizationTypeId: { type: String, required: true } },
-  methods   : { getOrganizationNameById },
+  props   : { organizationTypeId: { type: String, required: true } },
+  computed: {
+    ...mapGetters({ getTypeById: 'organizations/getTypeById' }),
```
Comment (sbilodeau) [inline components/list/columns/OrganizationType.vue:13]: Excellent! 😄

File: components/list/columns/OrganizationType.vue [hunk 4]
```diff
@@ -5,14 +5,18 @@
...
+  },
+  methods: { getOrganizationNameById },
 };
 
 function getOrganizationNameById(organizationTypeId) {
```
Comment (sbilodeau) [inline components/list/columns/OrganizationType.vue:18]: name is wrong `getOrganizationNameById`. it's not the organizationName but the organization**Type**Name

File: components/list/columns/OrganizationType.vue [hunk 5]
```diff
@@ -5,14 +5,18 @@
...
-  return this.$store.getters['organizations/typeNameById'](organizationTypeId);
+  const organization = this.getTypeById(organizationTypeId);
+  return organization ? organization.Title : '';
```
Comment (sbilodeau) [inline components/list/columns/OrganizationType.vue:20]: if type not found return the ID (help in debug)
also the variable name is wrong. should be `type` or `organizationType`
```suggestion
  const organizationType = this.getTypeById(organizationTypeId);
  return organizationType ? organizationType.Title : organizationTypeId;
```

File: assets/scss/_theme.scss (hunk 1 above)
Comment (sbilodeau) [reply assets/scss/_theme.scss:92]: please make sure you commit only lines which have changed

Comment (JamesSCBD) [reply assets/scss/_theme.scss]: Revert unchanged lines. I will make sure next time

File: components/list/OrganizationsList.vue (hunk 2 above)
Comment (JamesSCBD) [reply components/list/OrganizationsList.vue]: Done

File: components/list/columns/OrganizationType.vue (hunk 3 above)
Comment (JamesSCBD) [reply components/list/columns/OrganizationType.vue:13]: Thanks

File: components/list/columns/OrganizationType.vue (hunk 4 above)
Comment (JamesSCBD) [reply components/list/columns/OrganizationType.vue]: Rename method name to `getOrganizationTypeNameById`

File: components/list/columns/OrganizationType.vue (hunk 5 above)
Comment (JamesSCBD) [reply components/list/columns/OrganizationType.vue]: Done

**Pull request merged**